components/database/mariadb-101/Makefile

```diff
@@ -46,8 +46,8 @@ CMAKE_PLUGINDIR.64 =	lib/$(MACH64)/plugin
 
 CMAKE_OPTIONS+= -DCMAKE_C_COMPILER=$(CC)
 CMAKE_OPTIONS+= -DCMAKE_CXX_COMPILER=$(CXX)
-CMAKE_OPTIONS+= -DCMAKE_C_FLAGS=$(CFLAGS)
-CMAKE_OPTIONS+= -DCMAKE_CXX_FLAGS=$(CFLAGS)
+CMAKE_OPTIONS+= -DCMAKE_C_FLAGS="$(CFLAGS) "
+CMAKE_OPTIONS+= -DCMAKE_CXX_FLAGS="$(CFLAGS) "
 
 CMAKE_OPTIONS+= -DINSTALL_LAYOUT=SVR4
```

Comment:
Just a note: When you change something in a package without increasing its VERSION_NUMBER you'll need to bump its COMPONENT_REVISION. We have now a special situtation as I changed the RELEASE_MAJOR recently. So changing COMPONENT_REVISION is optional one time.